### PR TITLE
add test configuration for Travis

### PR DESCRIPTION
**Pull request by boegel**
depends on ~~https://github.com/hpcugent/easybuild-framework/pull/1733~~

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  apt:
+    packages:
+      - tcl8.5
+install:
+    # install easybuild-framework (and dependencies)
+    # use framework 'develop' branch, except when testing 'master'
+    - if [ "x$TRAVIS_BRANCH" = 'xmaster' ]; then FRAMEWORK_BRANCH=master; else FRAMEWORK_BRANCH=develop; fi
+    - easy_install https://github.com/hpcugent/easybuild-framework/archive/${FRAMEWORK_BRANCH}.tar.gz
+    # install environment modules tool using 'install_eb_dep.sh' script provided by easybuild-framework
+    - source $(which install_eb_dep.sh) modules-3.2.10 $HOME && source $MOD_INIT
```
Comment (wpoely86) [inline .travis.yml]: only test Tmod?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - 2.7
+addons:
+  apt:
+    packages:
+      - tcl8.5
+install:
+    # install easybuild-framework (and dependencies)
+    # use framework 'develop' branch, except when testing 'master'
+    - if [ "x$TRAVIS_BRANCH" = 'xmaster' ]; then FRAMEWORK_BRANCH=master; else FRAMEWORK_BRANCH=develop; fi
+    - easy_install https://github.com/hpcugent/easybuild-framework/archive/${FRAMEWORK_BRANCH}.tar.gz
```
Comment (wpoely86) [inline .travis.yml]: does it somewhere clearly show the commit of framework that is being used?

File: .travis.yml (hunk 1 above)
Comment (boegel) [inline .travis.yml]: For the easyblock unit tests, it doesn't matter at all, no module commands are run.

I'm even wondering now whether this is needed at all...
Maybe it needs to be there to make the framework happy.

File: .travis.yml (hunk 2 above)
Comment (boegel) [inline .travis.yml]: Hmm, good point... I'll check on that.

Comment (boegel) [inline .travis.yml]: OK, seems like it doesn't. I'll figure out the `git` (or `curl`) command we can use to determine the commit ID that corresponds to `develop` at that time.

**Comment by boegel** [comment]
@wpoely86 good to go, I think?

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**